src/main/kotlin/lotto/dto/Money.kt

```diff
@@ -0,0 +1,28 @@
+package lotto.dto
+
+import lotto.domain.LottoNumbers.Companion.LOTTO_PRICE
+
+class Money(money: Int) {
+
+    private val initialMoney = money
+    private var remainMoney = money
```

Comment:
```suggestion
class Money(private val initialMoney: Int) {
    private var remainMoney = initialMoney
```
생성자 프로퍼티를 사용할 수 있을 것 같습니다.